api-report/client-utils.api.md

```diff
@@ -120,7 +120,7 @@ export type TypedEventTransform<TThis, TEvent> = TransformedEvent<TThis, "newLis
 export function Uint8ArrayToArrayBuffer(array: Uint8Array): ArrayBuffer;
 
 // @internal
-export function Uint8ArrayToString(arr: Uint8Array, encoding?: string): string;
+export function Uint8ArrayToString(arr: Uint8Array, encoding?: "utf8" | "utf-8" | "base64"): string;
```

Comment:
An earlier version of a package that depends on this package, say internal-6.2.0, has a dependency range that will include this new version when we release it - unless the change is in next. Is this change truly safe in that light? If we expect to be able to make breaking changes in minor versions, even for "internal use only" packages, then we need to update our interdependency ranges to more accurately describe that.

Reply:
I've moved to 'next' as recommended.  Thanks!